include/deal.II/numerics/data_out.h

```diff
@@ -258,6 +258,23 @@ class DataOut : public DataOut_DoFData<DoFHandlerType,
    * In other words, using this parameter can not help you plot the solution
    * exactly, but it can get you closer if you use finite elements of higher
    * polynomial degree.
+   *
+   * @note Specifying `n_subdivisions>1` is useful when using higher order
+   *   finite elements, but in general it does not actually result in the
+   *   visualization showing higher order polynomial surfaces -- rather, you
+   *   just get a (bi-, tri-)linear interpolation of that higher order
+   *   surface on a finer mesh. However, when outputting the solution in the
+   *   VTK and VTU file formats via DataOutInterface::write_vtk() or
+   *   DataOutInterface::write_vtu() (where DataOutInterface is a base
+   *   class of the current class) as we often do in the tutorials,
+   *   you can provide a set of flags via the DataOutBase::VtkFlags
+   *   structure that includes the
+   *   DataOutBase::VtkFlags::write_higher_order_cells flag. When set, the
```

Comment:
We also need to remember to adjust this comment when VTU is given higher order output capabilities.

Reply:
I'm not sure I understand, but VtkFlags structure is common for both write_vtk and write_vtu...

Reply:
Yes, and the text is already written in anticipation that the VTU format will learn this as well.

Reply:
> I'm not sure I understand, but VtkFlags structure is common for both write_vtk and write_vtu...

Cool, well then I'm glad that there will be nothing to do here.